router/types/types.go

```diff
@@ -55,6 +55,7 @@ type JobMetadataT struct {
 	Secret             json.RawMessage `json:"secret"`
 	JobT               *jobsdb.JobT    `json:"jobsT"`
 	WorkerAssignedTime time.Time       `json:"workerAssignedTime"`
+	DestInfo           json.RawMessage `json:"destInfo,omitempty"`
```

Comment:
Where is this used?

Reply:
This information is being set by transformer and made use by transformer